staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types_jsonschema.go

```diff
@@ -196,6 +196,13 @@ type ValidationRule struct {
 	// If unset, the message is "failed rule: {Rule}".
 	// e.g. "must be a URL with the host matching spec.host"
 	Message string
+	// MessageExpression represents a CEL expression that can be evaluated when validation fails, whereas Message
```

Comment:
"MessageExpression declares a CEL expression that evaluates to the validation failure message that is returned when this rule fails" ?

Reply:
Right, fixed